templates/cpp-template-default/Classes/HelloWorldScene.h

```diff
@@ -38,7 +38,7 @@ class HelloWorld : public cocos2d::Scene
     void menuCloseCallback(cocos2d::Ref* pSender);
     
     // implement the "static create()" method manually
-    CREATE_FUNC(HelloWorld);
```

Comment:
I thought the semicolon is for the better readability?

Reply:
But it gives warning in editor (QtCreator) and thus have to manually remove it.

Reply:
Well, good to hear that. Then now this is the case I would say.